This is a test to see if you'll catch Â¿, but still beabel to finish after printing error message.
